<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDepSil</class>
 <widget class="QDialog" name="DialogDepSil">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Peer(s)</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color: #6F7F85;
}

QDialog{
background-color: white;
}

QListWidget::item:selected{
    background: #65B0C9;
}

QPushButton:focus{
	border: none;
	outline: none;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>310</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #BBC7C4;
color: #6F7F85;
font-size: 12pt;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: #6F7F85;
border: none;
font-size: 12pt;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#e8473c;&quot;&gt;Please note that the SIL- infrastructure will be completely reset on the selected peers and that all plugins and related data will be erased.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#e8473c;&quot;&gt;This Process can neither be undone nor stopped!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: #65B0C9;
background-color: white;
border-radius:6px;
border:1px solid #BBC7C4;
font-family:Ubuntu;
font-size:12pt;
}

QPushButton::hover{
color: white;
background-color: #65B0C9;
border-radius:6px;
border:1px solid #BBC7C4;
font-family:Ubuntu;
font-size:12pt;
}</string>
   </property>
   <property name="text">
    <string>Deploy SIL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
   border: none;
   color: #65B0C9;
}</string>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #65B0C9;</string>
   </property>
   <property name="text">
    <string>Peers</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
